src/sass/modules/_buttons.scss

```diff
@@ -132,17 +132,33 @@ $iconBtnSize: $btnHeight;
      &.active{
         color: #0039c6;
      }
+  
+    &:hover  {
+        text-decoration: none;
+        color: #0039c6;
+        outline: none;
+        background: none;
 
-     &:hover, &:focus  {
-         text-decoration: none;
-         color: #0039c6;
-         outline: none;
-         background: none;
+        & .value {
+            text-decoration: underline;
+        }
+    }
 
-         & .value {
-             text-decoration: underline;
-         }
-     }
+    &:focus {
+        text-decoration: none;
+        color: #0039c6;
+        outline: rgb(77, 144, 254) auto 5px;
+        outline-offset: -2px;
+
+        & .value {
+            text-decoration: underline;
+        }
+        
+        &:-moz-focusring:not(input):not(select):not(textarea):not(iframe):not(frame):not(body):not(html) {
```

Comment:
I'd like to know more about all these `:not()` selectors 😮

Reply:
Actually I just get it from line 713 of this document:
https://dxr.mozilla.org/mozilla-central/source/layout/style/res/html.css 
I deleted ":not(button)" for the buttons to be outlined, but I didn't touch others...:sweat_smile:

Reply:
Ah, I see! Nice doc reference. We shouldn't need the ":not" selectors since it's being applied to the .btn class.